addons/account_edi/models/account_edi_document.py

```diff
@@ -248,6 +249,10 @@ def _process_documents_web_services(self, job_count=None, with_commit=True):
             if with_commit and len(jobs_to_process) > 1:
                 self.env.cr.commit()
 
+        # Re-trigger fail-safe
+        all_jobs_blocking = all([all([d.blocking_level for d in job[0]]) for job in all_jobs])
```

Comment:
Fail quickly with generators instead of building the lists and computing/storing everything
```suggestion
        all_jobs_blocking = all(d.blocking_level for job in all_jobs for d in job[0])
```